scripts/ci-docker-images/builder-armv7-unknown-linux-musleabihf/Dockerfile

```diff
@@ -0,0 +1,332 @@
+ARG RUST_VERSION=1.38.0
```

Comment:
A lot of this seems repeated from our other musl container? I think this would be a good point to break this out into its own script?

Reply:
I want to combine this and the other one into a single Dockerfile when doing #946. For now the goal is to just have Vector compiled and running on the new platoform.

Reply:
Sounds good to me! I am just eager to get #946 done 😄